annif/templates/home.html

```diff
@@ -122,60 +137,96 @@ <h2 class="mt-4">Results</h2>\
 });
 
 // Application, which uses the components above
-var app = new Vue({
+new Vue({
   delimiters: ["<%","%>"],
   el: '#app',
   data: {
-    results: results_,
-    errors: []
+    text: '',
+    project: '',
+    projects: [],
+    results: [],
+    problems: [],
+    loading: false
```

Comment:
Now all the data of the app is centralized here in the `App`. I think it is easier to maintain it now, but would be even easier with Vuex later on.